emperor/support_files/js/controller.js

```diff
@@ -262,12 +263,40 @@ define([
       selector: '#' + scope.$divId.attr('id') + ' .emperor-plot-wrapper',
       trigger: 'none',
       items: {
+        'saveState': {
+          name: 'Save Emperor setup',
+          icon: 'edit',
+          callback: function(key, opts) {
+            scope.saveConfig();
+          }
+        },
+        'loadState': {
+          name: 'Load Emperor setup',
+          icon: 'load',
+          callback: function(key, opts) {
+            var file = $('<input id="myInput" type="file">');
```

Comment:
Can you change this? We can't really tie ourselves to having objects with non-unique identifiers. If the ID attribute is absolutely necessary, can we make it a randomly-generated one, if it is not necessary, can we just remove it?